pep-0634.rst

```diff
@@ -0,0 +1,706 @@
+PEP: 634
+Title: Structural Pattern Matching: Specification
+Version: $Revision$
+Last-Modified: $Date$
+Author: Brandt Bucher <brandtbucher@gmail.com>,
+        Guido van Rossum <guido@python.org>
+BDFL-Delegate:
+Discussions-To: Python-Dev <python-dev@python.org>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 12-Sep-2020
+Python-Version: 3.10
+Post-History:
+Replaces: 622
+Resolution:
+
+
+Abstract
+========
+
+TODO: Do we use "the subject matches the pattern" or "the pattern
+matches the subject"?
```

Comment:
Good question. The latter feels right to be, since our match protocol used "smart" patterns and "dumb" subjects.